src/base/bittorrent/session.h

```diff
@@ -330,6 +334,16 @@ namespace BitTorrent
         void setDiskCacheTTL(uint ttl);
         bool useOSCache() const;
         void setUseOSCache(bool use);
+        bool guidedReadCache() const;
```

Comment:
Please use naming convention for bool properties: isGuidedReadCacheUsed or isGuidedReadCacheEnabled.
The same comment is for other bool properties.